Tests/ComposableArchitectureTests/ReducerBuilderTests.swift

```diff
@@ -196,7 +196,7 @@ private struct ForEachExample {
   }
 
   enum Action {
-    case value(IdentifiedAction<Element.ID, Never>)
+    case value(IdentifiedAction<Int, Never>)
```

Comment:
`@CasePathable` getting more complex means a new macro bug where it doesn't see the `.ID` in time 😕